src/wrappers/formly-wrappers-bootstrap-addons.html

```diff
@@ -0,0 +1,11 @@
+<div ng-class="{'input-group': to.addonLeft || to.addonRight}">
+    <div class="input-group-addon" ng-if="to.addonLeft">
+        <span ng-class="'{{to.addonLeft.class}}'" ng-if="to.addonLeft.class"></span>
```

Comment:
Instead of using `ng-class` I would recommend simply doing `class="{{to.addonLeft.class}}"`. Also, I'm not certain what this span does. If I specify a class, the span appears, but it has no contents, so does this actually do anything?